packages/sdk/src/job.ts

```diff
@@ -0,0 +1,39 @@
+import { get } from './request';
+import { tunaMirrorURI } from './utils';
+
+export class Job {
+  route: string;
+  constructor(host: string, port: number) {
+    this.route = `${host}:${port}/job`;
```

Comment:
As we have created a `API` or `PipcookClient`, I think it's good to share the `host` and `port` there.